openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -601,6 +601,7 @@ def _get_representations(self, instance, exp_files):
                 "files": os.path.basename(remainder),
                 "stagingDir": os.path.dirname(remainder),
             }
+            representations.append(rep)
```

Comment:
Interesting - was this just a bug that it was missing before? Seems like something that should've be found earlier :D